### PR TITLE
COUCHDB-2794: Docs not fully correct for couch_httpd_auth/secret

### PR DESCRIPTION
**Pull request by andywenk**
rewording and added also the link to Cookie Auth section.

### CODE REVIEW

File: src/config/auth.rst [hunk 1]
```diff
@@ -214,7 +214,7 @@ Authentication Configuration
 
     .. config:option:: secret :: Proxy Auth secret token
 
-        The secret token used for :ref:`api/auth/proxy` method. ::
+        The secret token is used for :ref:`api/auth/proxy` and for :ref:`api/auth/cookie`. ::
```
Comment (kxepal) [inline src/config/auth.rst]: Thanks, but look on one line above (;

Comment (andywenk) [inline src/config/auth.rst]: sry I don't know what you mean

Comment (kxepal) [inline src/config/auth.rst]: Line 215: `.. config:option:: secret :: Proxy Auth secret token`

Comment (andywenk) [inline src/config/auth.rst]: do you want sth like "Proxy and Cookie Auth" in line 215 ?

Comment (kxepal) [inline src/config/auth.rst]: I think about just generic "authentication secret token" or something, but you may figure something else and better (:

Comment (andywenk) [inline src/config/auth.rst]: I like that ;-)

**Pull request closed without merging**